packages/base/src/panelview/formbuilder.tsx

```diff
@@ -151,6 +151,12 @@ export class ObjectPropertiesForm extends React.Component<IProps, IStates> {
         <div
           className="jpcad-property-panel"
           data-path={this.props.filePath ?? ''}
+          // Prevent Delete/Backspace from propagating to other elements
+          onKeyDown={(e: React.KeyboardEvent) => {
+            if (e.key === 'Delete' || e.key === 'Backspace') {
```

Comment:
Should we go further and prevent the propagation of all keyboard events? I feel like typing in that panel shouldn't be disrupted by other keyboard shortcuts 

@trungleduc do you have an opinion?